### ISSUE TITLE
Channel prefix notation is not considered

### ISSUE THREAD

**Issue by johanneskoester** [QuantStack/mamba#209]
The command

```
mamba create -c conda-forge -n snakemake bioconda::snakemake bioconda::snakemake-minimal
```
Does not find the snakemake and snakemake-minimal packages, which are in the bioconda channel.
This works fine with conda though.

**Comment by davidbrochart** [comment]
There was some initial work on it (https://github.com/QuantStack/mamba/pull/114), but we need to find a better solution.

**Comment by wolfv** [comment]
@johanneskoester I am actually wondering what behavior you'd prefer for this case:

Select only bioconda::snakemake & bioconda::snakemake-minimal and otherwise use otherwise active channels, or to implicitly add bioconda as the highest priority channel?

**Comment by wolfv** [comment]
fixed now in #277
